version/version.go

```diff
@@ -0,0 +1,22 @@
+package version
+
+import "runtime/debug"
+
+var version = func() string {
+	bi, ok := debug.ReadBuildInfo()
+	if !ok {
+		return "unknown"
```

Comment:
I've not been able to avoid getting "unknown" when I import stanza into OpenTelemetry. It's possible that this is just my IDE, but I can't think of any specific reason why that would be the case.

Have you had any luck with similar, or have any ideas on how to validate that this works in an imported context?

Reply:
Very interesting. `stanza version` works when I build this, and the stanza binary imports the module (I think) the same way as OT does. I'll see if I can reproduce what you're seeing in OT

Reply:
@djaglowski Seemed to work fine for me when building the OT binary. Was this in a test or something? 
```2020/10/05 15:37:27 Running stanza version v0.12.1```

Reply:
@camdencheek Thanks for confirming. I did try this in a test. Is there a reason you'd expect it not to work there?

Reply:
@djaglowski No clear reason. It was just kinda a guess since the runtime environment of tests is probably different the runtime of a binary